src/main/java/bhs/devilbotz/subsystems/Gripper.java

```diff
@@ -52,5 +59,13 @@ public void stop() {
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    System.out.println(
+        pcmCompressor.getPressure() + "\n " + pcmCompressor.getPressureSwitchValue());
```

Comment:
We should get rid of this and put it on the dashboard, instead.

Reply:
Get pressure does not work, I forgot to remove it before committing. Will remove now